tests/unit/slider/methods.js

```diff
@@ -109,8 +109,54 @@ test( "value", function() {
 	equal( element.slider( "value" ), 460, "value is restricted to maximum valid step" );
 });
 
-//test( "values", function() {
-//	ok(false, "missing test - untested code is broken code." );
-//});
+test( "values", function() {
+	expect( 12 );
+	
+	var element = $( "<div></div>" ).slider({
+		range: true,
+		min: 10,
+		max: 100,
+		step: 1
+	});
+
+	element.slider( "values", [ 20, 90 ] );
```

Comment:
All of these were addressed by #1573, which landed in 4c5cd2018d88392871cfc93d3fb6f71a0bc510ff